### PR TITLE
Data model changes for User table.

### PR DESCRIPTION
**Pull request by prashbnair**
Data model changes for User table.

### CODE REVIEW

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
Please squash your commits once all is done. Also im hoping that its all tested on dev cluster

File: f8a_worker/models.py [hunk 1]
```diff
@@ -401,6 +401,19 @@ class RecommendationFeedback(Base):
...
+    """Table for user details."""
+
+    __tablename__ = "user_details"
+
+    user_id = Column(String(64), primary_key=True)
+    snyk_api_token = Column(String(64))
+    last_validated_date = Column(DateTime)
+    status = Column(String(32))
+    registered_date = Column(DateTime)
+    created_date = Column(DateTime)
```
[the remark this answers is not in the document]
Comment (prashbnair) [reply f8a_worker/models.py:414]: Done

**Comment by yzainee** [comment]
[test]

**Comment by yzainee** [comment]
[test]

**Pull request merged**